ankaios_sdk/ankaios.py

```diff
@@ -0,0 +1,733 @@
+            manifest (Manifest): The manifest object to be deleted.
+
+        Returns:
+            bool: True if the manifest was deleted successfully,
+                False otherwise.
+        """
+        request = Request(request_type="update_state")
+        request.set_complete_state(CompleteState())
+        request.set_masks(manifest._calculate_masks())
+
+        # Send request
+        try:
+            response = self._send_request(request)
+        except TimeoutError as e:
+            self.logger.error("%s", e)
+            return False
+
+        # Interpret response
+        (content_type, content) = response.get_content()
+        if content_type == "error":
+            self.logger.error("Error while trying to delete manifest: %s",
+                              content)
+        elif content_type == "update_state_success":
+            self.logger.info(
+                "Update successfull: %s added workloads, "
+                + "%s deleted workloads.",
+                content["added_workloads"], content["deleted_workloads"]
+            )
+            return True
+        return False
+
+    def run_workload(self, workload: Workload) -> bool:
+        """
+        Send a request to run a workload.
+
+        Args:
+            workload (Workload): The workload object to be run.
+
+        Returns:
+            bool: True if the workload was run successfully, False otherwise.
+        """
+        complete_state = CompleteState()
+        complete_state.set_workload(workload)
+
+        # Create the request
+        request = Request(request_type="update_state")
+        request.set_complete_state(complete_state)
+        request.set_masks(workload.masks)
+
+        # Send request
+        try:
+            response = self._send_request(request)
+        except TimeoutError as e:
+            self.logger.error("%s", e)
+            return False
+
+        # Interpret response
+        (content_type, content) = response.get_content()
+        if content_type == "error":
+            self.logger.error("Error while trying to run workload: %s",
+                              content)
+        elif content_type == "update_state_success":
+            self.logger.info(
+                "Update successfull: %s added workloads, "
+                + "%s deleted workloads.",
+                content["added_workloads"], content["deleted_workloads"]
+            )
+            return True
+        return False
+
+    def delete_workload(self, workload_name: str) -> bool:
+        """
+        Send a request to delete a workload.
+
+        Args:
+            workload_name (str): The name of the workload to be deleted.
+
+        Returns:
+            bool: True if the workload was deleted successfully,
+                False otherwise.
+        """
+        request = Request(request_type="update_state")
+        request.set_complete_state(CompleteState())
+        request.add_mask(f"desiredState.workloads.{workload_name}")
```

Comment:
It would be better to have a variable that is used everywhere with this. The question is where to put it?
Currently I don't have an "utils.py" script with functionality needed by all the classes in the sdk. And also I don't have a general configuration file, to put it there.

Reply:
Then, you now have the perfect excuse to create a `utils.py` or even better `constantants.py` 🤣 as we don't have any utils yet.

Reply:
Done. Created a `utils.py` to be able to put general methods inside as well (if needed).